src/views/Community/index.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react'
+// import Dashboard from '../Dashboard'
+import { connect } from 'react-redux'
+import { compose } from 'redux'
+import { sendCommunityData } from '../../store/actions/community'
+import { Card,Col, Row, Progress } from 'reactstrap';
+import { Link } from 'react-router-dom'
+import { isAuthUserType } from '../../services/Auth'
+
+
+class Community extends Component {
+  constructor(props) {
+    super(props)
+
+    this.deleting = this.deleting.bind(this)
+  }
+
+  componentDidMount() {
+    let { sendData, communityData } = this.props
+    if (communityData.communityData.length === 0) {
+      sendData('list', 'get');
+    } else {
+      if (communityData.communityData.data.length < 2) {
+        sendData('list', 'get');
+      }
+    }
+  }
+
+  deleting(id) {
+    let formData = new FormData();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['formData' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)